website/source/docs/configuration/storage/google-cloud.html.md

```diff
@@ -34,11 +34,16 @@ storage "gcs" {
   account must have permission to read, write, and delete from the bucket. This
   can also be provided via the environment variable `GOOGLE_STORAGE_BUCKET`.
 
-- `credentials_file` `(string: <required>)` – Specifies the path on disk to a
+- `credentials_file` `(string: "<varies>")` – Specifies the path on disk to a
   Google Cloud Platform [service account][gcs-service-account] private key file
   in [JSON format][gcs-private-key]. This can also be provided via the
```

Comment:
If we make the change to have `GOOGLE_APPLICATION_CREDENTIALS` just come from the ADC client init, we should also remove the sentence:
```This can also be provided via the environment variable `GOOGLE_APPLICATION_CREDENTIALS`.```

Then the user would either set `credentials_file` or refer to the ADC docs

Reply:
See my other comment. I think we should keep this because it's the standard way that users of gcloud SDK would be familiar with it.